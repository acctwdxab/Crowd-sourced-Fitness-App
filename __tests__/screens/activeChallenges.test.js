import React from 'react';
import renderer, { act } from 'react-test-renderer';

import ActiveChallenges from '../../src/screens/activeChallenges';

describe('<ActiveChallenges /> load', () => {
    it('has 1 child', () => {
        const tree = renderer.create(<ActiveChallenges />).toJSON();
        expect(tree.children.length).toBe(1);
    });

    it('renders correctly', async () => {
        const tree = renderer.create(<ActiveChallenges />).toJSON();
        await act(async () => { await expect(tree).toMatchSnapshot(); })
    });
});