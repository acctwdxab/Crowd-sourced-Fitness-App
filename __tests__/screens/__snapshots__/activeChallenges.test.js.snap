// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ActiveChallenges /> load renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "white",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Active Challenges
  </Text>
</View>
`;
